Doc/whatsnew/2.2.rst

```diff
@@ -171,22 +171,23 @@ attributes of their own:
 
 * :attr:`~definition.__name__` is the attribute's name.
 
-* :attr:`!__doc__` is the attribute's docstring.
+* :attr:`~definition.__doc__` is the attribute's docstring.
 
-* ``__get__(object)`` is a method that retrieves the attribute value from
+* :meth:`~object.__get__` is a method that retrieves the attribute value from
```

Comment:
Prefer to leave this unchanged, since this is an old What's New and the text displayed to users is changing. I would edit old What's News only to fix errors/warnings and to fix egregious mistakes that cause user confusion.